packages/sitecore-jss-react/src/components/Link.test.tsx

```diff
@@ -140,21 +139,77 @@ describe('<Link />', () => {
       rawValue: 'Test1',
     };
 
-    const field = {
-      href: '/lorem',
-      text: 'ipsum',
-      metadata: testMetadata,
-    };
-    const rendered = mount(<Link field={field} />);
-
-    expect(rendered.html()).to.equal(
-      [
-        `<code type="text/sitecore" chrometype="field" class="scpm" kind="open">${JSON.stringify(
-          testMetadata
-        )}</code>`,
-        '<a href="/lorem">ipsum</a>',
-        '<code type="text/sitecore" chrometype="field" class="scpm" kind="close"></code>',
-      ].join('')
-    );
+    it('should render field metadata component when metadata property is present', () => {
+      const field = {
+        href: '/lorem',
+        text: 'ipsum',
+        metadata: testMetadata,
+      };
+      const rendered = mount(<Link field={field} />);
+
+      expect(rendered.html()).to.equal(
+        [
+          `<code type="text/sitecore" chrometype="field" class="scpm" kind="open">${JSON.stringify(
+            testMetadata
+          )}</code>`,
+          '<a href="/lorem">ipsum</a>',
+          '<code type="text/sitecore" chrometype="field" class="scpm" kind="close"></code>',
+        ].join('')
+      );
+    });
+
+    it('should render default empty field placeholder when field value is empty in edit mode metadata', () => {
+      const field = {
+        value: '',
```

Comment:
Need to add a test case to cover: when field is the value itself (GenericFIeldValue) type (applicable for rest of unit tests below)